cogs/temp_channels.py

```diff
@@ -100,82 +94,109 @@ async def on_raw_reaction_add(self, payload: RawReactionActionEvent):
+            await TmpChannelUtil.joined_voice_channel(self.db, temp_channels, after.channel, guild,
+                                                      member, self.bot, self.join_db)
 
     @group(pass_context=True,
-           name="studyChannel",
+           name="tempChannel",
            help="Configure study channel settings.")
     @bot_chat(bot_channels)
     @has_guild_permissions(administrator=True)
-    async def study_channel(self, ctx: Context):
+    async def temp_channel(self, ctx: Context):
         if not ctx.invoked_subcommand:
             raise BadArgument
         member: Union[Member, User] = ctx.author
         logger.info(f'User="{member.name}#{member.discriminator}({member.id})", Command="{ctx.message.content}"')
 
-    @study_channel.command(pass_context=True,
-                           name="category",
-                           brief="Sets the category.",
-                           help="The tmp channels will be found under this category.\n"
-                                "The category must be given as an int value.")
-    async def study_channel_category(self, ctx: Context, category: int):
+    @temp_channel.command(pass_context=True,
+                          name="join-add",
+                          brief="Sets a join channel.",
+                          help="Joining the chosen channel will create a tmp channel.\n"
+                               "The channel must be given as an int value.")
+    async def study_channel_join_add(self, ctx: Context, channel: VoiceChannel, pattern: str, persistent: bool):
         """
-        Saves the category of the tmp study channels:
+        Saves enter point of tmp study channels:
 
         Args:
             ctx: The command context provided by the discord.py wrapper.
 
-            category: The category id for the channels.
-        """
+            channel: The voice_channel id.
+
+            pattern: The pattern of the default name: Example: Study-{0:02d}.
 
-        db = PrimitiveMongoData(CollectionEnum.CATEGORIES)
-        key = ConfigurationNameEnum.STUDY_CATEGORY
-        msg = "category"
-        await TmpChannelUtil.update_category_and_voice_channel(category, ctx, db, key, msg)
-
-    @study_channel.command(pass_context=True,
-                           name="join",
-                           brief="Sets a join channel.",
-                           help="Joining the chosen channel will create a tmp channel.\n"
-                                "The channel must be given as an int value.")
-    async def study_channel_join(self, ctx: Context, channel: int):
+            persistent: The possibility that the created channel may persist.
         """
-        Saves the enter point of tmp study channels:
+        await self.join_db.insert_one((channel, pattern, persistent))
+        indicator = "" if persistent else "none"
+        await ctx.reply(f"You can now create {indicator} persistent channels with {channel.category}:{channel.name}.")
```

Comment:
Instead of `channel.name` use `channel.mention`

Reply:
Will chnage that.

Reply:
Done, please wait for push.